client/packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -90,6 +93,7 @@ export function Autocomplete<T>({
           disableUnderline: false,
           sx: {
             padding: '4px !important',
+            ...textSx != null ? textSx : {},
```

Comment:
The `textSx` should never be `null` as you've defined it as `textSx?: SxProps`, so either `SxProps` or undefined.
Also, the lack of brackets is confusing, as it's not clear you're spreading the whole thing, not just the first `textSx` (I don't actually know what the default order of operations would be here). 

But I think the cleanest approach is to just have `...textSx` here, and then in the function input give it a default of `{}`.